src/tutorial/introduction.rst

```diff
@@ -93,9 +86,11 @@ Cylc can automatically:
 .. nextslide::
 
 Cylc provides a variety of command line and GUI tools for visualising and
-interacting with workflows.
+interacting with workflows. Shown below are Cylc TUI (Text User Interface),
+GUI and ``cylc scan``.
 
-.. image:: img/cylc-gui.png
+.. image:: /tutorial/img/cylc-tools.png
```

Comment:
Preceding `/` on purpose?

Reply:
I struggled to get it working so i just gave it the full path.